packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -187,6 +189,12 @@ contract OPContractsManager is ISemver, Initializable {
     /// @notice Thrown when the latest release is not set upon initialization.
     error LatestReleaseNotSet();
 
+    /// @notice Thrown when the starting anchor roots are not provided.
+    error InvalidStartingAnchorRoots();
+
+    /// @notice Thrown when the salt mixer is not provided.
+    error InvalidSaltMixer();
```

Comment:
and if above suggestion is applied we can also delete this
```suggestion
```

Reply:
https://github.com/ethereum-optimism/optimism/pull/12152/commits/012cdf54a9504bd6b62f9e6eab4acce452642193